tslint/tslint_annotations.js

```diff
@@ -0,0 +1,45 @@
+// Copyright 2019 VMware, Inc.
+// SPDX-License-Identifier: BSD-2-Clause
+
+const { getAnnotationLevel } = require('../annotations_levels')
+
+/**
+ * Translates the tslint severity from "ERROR" | "WARNING" | "OFF"
+ * to the "HIGH", "MEDIUM" and "LOW"
+ * @param {String} severity issues severity given by tslint
+ */
+function translateSeverity (severity) {
```

Comment:
Aren't we doing similar mappings like this in annotations_levels.js?

Reply:
We don't have to.
The severity attribute in the output from Bandit and Gosec is "HIGH", "MEDIUM" or "LOW" like the ones used in the annotations_levels.js.
This is a specific situation in this linter.

But there is another annoying thing. There is no "confidence" in the output and I give "HIGH" confidence by default...

Reply:
So yes, `annotations_levels.js` looks at confidence whereas TSLint has none, but that's a very easy change to use a `default` in the switch statement for cases where there is no confidence.

Also, looks odd to map `OFF` to `LOW` since I assume `OFF` indicates you'll get no results.  Although I'm not clear what happens with `OFF` according to the documentation.

Reply:
So we should ignore the issues with annotations off?

I will speak with the TSLint team about it.

Reply:
When there is no confidence from a linters output what should be the default value? HIGH?